### PR TITLE
Add measurement log address and index to FHT.

### PR DESCRIPTION
**Pull request by bluegate010**

### CODE REVIEW

**Review by mhatrevi — APPROVED** [review]

File: rom/dev/src/fht.rs [hunk 1]
```diff
@@ -190,6 +191,9 @@ pub fn make_fht(env: &RomEnv) -> FirmwareHandoffTable {
         pcr_log_addr,
         // TODO: remove `pcr_bank.log_index` and just increment `fht.pcr_log_index`
```
Comment (mhatrevi) [inline rom/dev/src/fht.rs:192]: Can we address these TODOs in this PR as well?

**Review by mhatrevi — CHANGES_REQUESTED** [review]

File: rom/dev/src/fht.rs (hunk 1 above)
Comment (bluegate010) [reply rom/dev/src/fht.rs:192]: Will be easier to do that if we can get https://github.com/chipsalliance/caliptra-sw/pull/779 in first.

Comment (bluegate010) [reply rom/dev/src/fht.rs:192]: Alternatively, if this goes in first, #779 will be amended to resolve this todo (already resolved the first todo).

File: fmc/README.md [hunk 2]
```diff
@@ -137,6 +137,8 @@ fields may not be changed or removed). Table revisions with different Major Vers
 | fmcalias_tbs_size     | 2            | ROM        | FMC Alias TBS Size.                                                                                      |
 | pcr_log_addr          | 4            | ROM        | PCR Log Address.                                                                                         |
 | pcr_log_index         | 4            | ROM        | Last empty PCR log entry slot index.                                                                     |
+| meas_log_addr         | 4            | ROM        | Measurement Log Address.                                                                                 |
+| meas_log_index        | 4            | ROM        | Last empty Measurement log entry slot index.                                                             |
```
[the remark this answers is not in the document]
Comment (bluegate010) [reply fmc/README.md:141]: Done.

File: rom/dev/src/fht.rs (hunk 1 above)
Comment (bluegate010) [reply rom/dev/src/fht.rs]: New TODO is gone.

**Review by mhatrevi — APPROVED** [review]

**Review by korran — APPROVED** [review]

**Review by FerralCoder — APPROVED** [review]

**Pull request merged**